sugarscape.py

```diff
@@ -970,6 +972,12 @@ def verifyConfiguration(configuration):
     if configuration["agentTagStringLength"] > 0 and configuration["environmentMaxTribes"] > configuration["agentTagStringLength"]:
         configuration["environmentMaxTribes"] = configuration["agentTagStringLength"]
 
+    configuration["agentDecisionModelTribalFactor"].sort()
```

Comment:
We should do a sort of every config option that represents a range of values near the top of `verifyConfiguration`. @amuller26, create a Kanban card for that and add that to your plate.

Reply:
Just make sure you keep in mind the config options that can have `[positive number, -1]` like the pollution timeframes.

Reply:
Exactly. There are edge cases to keep in mind.